lib/services/systemd.c

```diff
@@ -701,10 +698,7 @@ systemd_unit_exec_with_unit(svc_action_t * op, const char *unit)
         }
 
     } else {
-        DBusError error;
-
-        reply = systemd_send_recv(msg, &error, op->timeout);
-        dbus_error_free(&error);
+        reply = systemd_send_recv(msg, NULL, op->timeout);
```

Comment:
with the error logging removed from pcmk_dbus_find_error wouldn't this lead to no error logged at all for this case?